Update visual-diff README with new AWS setup instructions (#34)
BrightspaceUI/actions: visual-diff/README.md

## Setting Up AWS Access Creds

In order to have the `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and `AWS_SESSION_TOKEN` available to you, you will need to add that info to your repo setup in [`repo-settings`](https://github.com/Brightspace/repo-settings).  The role you want to assume already exists, so you only need to worry about "signing up" to receive rotating tokens for it.

Specifically, you will need to add the following to `<your_repo>.yaml` in [https://github.com/Brightspace/repo-settings/tree/main/repositories/github](https://github.com/Brightspace/repo-settings/tree/main/repositories/github):
```
aws_access:
  repo:
    assumable_role_arns:
      # Dev-UiPlatform
      - arn:aws:iam::037018655140:role/visual-diff-githubactions-access
```

Once you've merged the above, you'll see the new secrets in your repo within a few minutes!  These tokens will be rotated automatically for you.

Notes:

> * This uses [`iam-build-tokens`](https://github.com/Brightspace/iam-build-tokens) hub roles behind the scenes, in case your repo needs (or will one day need) access to multiple roles.

Reviewer: This supports multiple roles as well. Just that the role would be changing / recreated in the migration, and I didn't want to assess the assume role policies of the ones I didn't migrate.

Reviewer: Maybe

```suggestion
* This uses [`iam-build-tokens`](https://github.com/Brightspace/iam-build-tokens) hub roles behind the scenes, in case your repo is already configured with a hub-role there.
```

Author: Right, I was just trying to point out that this also supports multiple roles.  Is what's there now better?

Reviewer: Oh okay. I interpreted it as "if you need multiple roles, you can always head over to iam-build-tokens".

All good. Doesn't matter much to me right now 😄
